teslausb-www/html/index.html

```diff
@@ -912,20 +928,27 @@
             <div class="dropdownitem layoutitem" onclick="setLayout(3)">Side and rear on bottom, looking back</div>
             <div class="dropdownitem layoutitem" onclick="setLayout(4)">Mobile layout,<br>mirrored</div>
             <div class="dropdownitem layoutitem" onclick="setLayout(5)">Mobile layout,<br>looking back</div>
+            <div class="dropdownitem layoutitem" onclick="setLayout(6)">map and info on top, mirrored back</div>
```

Comment:
This is the only view that shows pillar cams, yet the description doesn't mention this at all. Might also be good to hide this option if no pillar recordings are available.